packages/libs/src/sdk/middleware/addAppNameMiddleware.ts

```diff
@@ -47,11 +47,14 @@ export const addAppNameMiddleware = ({
       }
 
       return {
+        ...context,
```

Comment:
just curious, what else is in `context`?

Reply:
the fetch API itself, it's weird, but felt lossy not to include that spread